rust/src/nfs/nfs.rs

```diff
@@ -1112,7 +1228,8 @@ impl NFSState {
                     // go to the next record.
                     match parse_rpc(cur_i, true) {
                         Ok((_, ref rpc_record)) => {
-                            self.process_request_record(rpc_record);
+                            self.add_rpc_tcp_ts_creds(flow, stream_slice, &cur_i[28..], rpc_record.creds_len as i64);
```

Comment:
I think 28 should be a named constant

Reply:
Lets pass the + 8 (also with named constant) in here as well instead of doing that calc in the function

Reply:
> 

should I go with a generic name? e.g `&cur_i[x..]`

Reply:
Ok so this would be for both `rpc_tcp` | `rpc_udp`, would something like `RPC_TCP_PRE_CREDS` do it?

Reply:
are the values the same for both tcp and udp?

Reply:
no, its 28 for tcp and 24 for udp

Reply:
I meant two constants with a format like `RPC_TCP_PRE_CREDS` | `RPC_UDP_PRE_CREDS`

Reply:
yes sounds good